scripts/make-packages.sh

```diff
@@ -182,6 +183,7 @@ if [ "$OS" == "linux" ]; then
     --provides ${NAME} \
     --conflicts ${CONFLICTS} \
     --depends "libssl${TLSVERSION}" \
+    --depends "libnuma1" \
```

Comment:
@wfurt I did a combination of guessing and chatgpt to come up with these changes. Would it be possible to get your help validating?

Reply:
```
[root@061eeb412ce3 /]# yum provides libnuma.so.1
Last metadata expiration check: 0:04:56 ago on Fri Jun  2 22:12:46 2023.
numactl-libs-2.0.16-2.fc38.i686 : libnuma libraries
Repo        : fedora
Matched from:
Provide    : libnuma.so.1
```

```
root@f45c0dbd52df:~/msquic# apt-cache depends libnuma1
libnuma1
  Depends: libc6
```

Looks good to me at least. Both dependency names can be recognized.